### PR TITLE
ISSUE-105 Upgrade to use USGS Spectral Library Version 7

### PR DESCRIPTION
**Pull request by bdegley4789**

### CODE REVIEW

**Comment by lewismc** [comment]
@bdegley4789 please merge master branch into this PR and push. This is why the tests are failing. Thank you.

**Comment by bdegley4789** [comment]
@lewismc ok, I have merge your changes to the docker file. Hopefully that fixes the ci/dockcloud

File: examples/example_spectral07.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+@contact:    coal-capstone@googlegroups.com
+'''
+
+#!/usr/bin/python
+
+import os
+import sys
+import pycoal
+from sys import path
```
Comment (lewismc) [inline examples/example_spectral07.py]: Remove this import, you've already imported ```sys``` and then use it in line 44 as per ```sys.path.insert(0, '../pycoal')```

**Review by lewismc — CHANGES_REQUESTED** [review]
@bdegley4789 by introducing ```importlib`` you are breaking the build... see the tests
```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/build/capstone-coal/pycoal/pycoal/tests/mineral_test.py", line 25, in <module>
    from pycoal import mineral
  File "/home/travis/build/capstone-coal/pycoal/pycoal/mineral.py", line 29, in <module>
    importlib.reload(sys)
AttributeError: 'module' object has no attribute 'reload'
```
Please fix and address comments, thanks.

**Comment by lewismc** [comment]
@bdegley4789 please always [test your code locally](https://github.com/capstone-coal/pycoal#tests) before submitting a PR.

**Comment by bdegley4789** [comment]
Yes, sorry about that. I'm working on my work computer right now and don't have all the dependencies for pycoal installed. 
I can't remember why I have importlib.reload(sys) but I recall adding it to get something to work. I'll test later when I get back home

**Comment by lewismc** [comment]
> Yes, sorry about that...

No problem, I'm just trying to make your life a bit easier rather than you always seeing the failing CI builds for your pull requests.

**Comment by bdegley4789** [comment]
nosetests now passes
screen shot 2018-04-18 at 10 55 58 pm

**Comment by bdegley4789** [comment]
I think travis-ci wants tests for the classes I created in mineral.py
I started on that yesterday. Will submit either tomorrow or friday
I'll close this pr for now since travis-ci is still failing

**Comment by lewismc** [comment]
@bdegley4789 any updates here/ Thanks

**Comment by bdegley4789** [comment]
@lewismc I'm still doing the tests for my conversion code, I'll have a new pull request with tests of my code up tonight or tomorrow. I also started a mineral classification with s07 envi .sli and .hdr on our default example on Sunday, it is still running

**Comment by lewismc** [comment]
ACK ok thank you for the update I appreciate it.

**Pull request closed without merging**